src/cljs/analysis_viewer/map_emitter.cljs

```diff
@@ -136,21 +145,28 @@
                                                 :count-attr (get attributes :count)}
                                                (calc-show-percs point)))))))
         transitions-objects (->> lines
-                                 (map (fn [{:keys [startPointId endPointId attributes] :as line}]
+                                 (keep (fn [{:keys [startPointId endPointId attributes] :as line}]
                                         (let [start-point (get points-index startPointId)
                                               end-point (get points-index endPointId)
                                               start-loc (point-location (:id start-point))
                                               end-loc   (point-location (:id end-point))
                                               from-coord (calc-proj-coord (:coordinate start-loc))
                                               to-coord (calc-proj-coord (:coordinate end-loc))]
-                                          (merge
-                                           {:type :transition
-                                            :from-coord from-coord
-                                            :to-coord to-coord
-                                            :from-label (:id start-loc)
-                                            :to-label   (:id end-loc)
-                                            :attrs attributes}
-                                           (calc-show-percs line))))))
+
+                                          ;; there is data that contains zero length transitions
+                                          ;; remove it here since it doesn't make sense
+                                          (if (zero? (math-utils/distance from-coord to-coord))
+
+                                            (js/console.warn "Skipping zero length transition" (str line))
```

Comment:
info?

Reply:
I prefer to have it as a warning since from the [Viewer] perspective it is skipping because the data is wrong, so it is warning the user that is not going to render some data it got because of something wrong in it. I think we should fix it in the analyzer (some day :P)